src/axom/sidre/examples/CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ blt_list_append(TO sidre_example_depends ELEMENTS mpi IF MPI_FOUND)
 if(MFEM_FOUND AND AXOM_ENABLE_MFEM_SIDRE_DATACOLLECTION)
     list(APPEND example_sources sidre_mfem_datacollection_vis.cpp)
```

Comment:
nitpick: I know this was here before, but you dont have to call list(APPEND ..) individually for each element, you can just add them with a space delimiter:

```
list(APPEND sources 
     foo.cpp
     bar.cpp
     baz.cpp)
```